cmd/docs/docs.go

```diff
@@ -0,0 +1,76 @@
+package docs
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/cobra/doc"
+)
+
+type Options struct {
+	Path string
+}
+
+const DefaultPath = "./docs"
+
+func DeterminePath(args []string) (string, error) {
+	if len(args) == 0 {
+		fmt.Printf("No path was provided. Using default path: %s\n", DefaultPath)
+		return DefaultPath, nil
+	}
+
+	absPath, err := filepath.Abs(args[0])
+
+	if err != nil {
+		return "", err
+	}
+
+	return absPath, nil
```

Comment:
This chunk can be simplified to:

```suggestion
	return filepath.Abs(args[0])
```